packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -538,11 +538,13 @@ export default class NodeResolver {
         let moduleDir = path.join(dir, 'node_modules', moduleName);
         let stats = await this.fs.stat(moduleDir);
         if (stats.isDirectory()) {
+		      const realModuleDir = this.options?.inputFS.realpathSync(moduleDir);
```

Comment:
```suggestion
          const realModuleDir = this.fs.realpathSync(moduleDir);
```

OK found it. At some point, the names of the variables changed. That's why `this.options` does not exist anymore.